externs/browser/gecko_dom.js

```diff
@@ -145,57 +149,39 @@ Window.prototype.updateCommands;
 /**
  * @see https://developer.mozilla.org/en/DOM/document.alinkColor
  * @type {string}
+ * @deprecated
  */
 Document.prototype.alinkColor;
 
 /**
  * @see https://developer.mozilla.org/en/DOM/document.anchors
  * @type {HTMLCollection<!HTMLAnchorElement>}
+ * @deprecated
  */
 Document.prototype.anchors;
 
 /**
  * @see https://developer.mozilla.org/en/DOM/document.applets
  * @type {HTMLCollection<!HTMLAppletElement>}
+ * @deprecated
  */
 Document.prototype.applets;
 /** @type {?string} */ Document.prototype.baseURI;
 
 /**
  * @see https://developer.mozilla.org/en/DOM/document.bgColor
  * @type {string}
+ * @deprecated
  */
 Document.prototype.bgColor;
 
-/** @type {HTMLBodyElement} */ Document.prototype.body;
```

Comment:
Hello Zbynek. Both 'body' and 'cookie' are required Document properties. I ran a presubmit and I found issues cause by those two properties.

Did you intend to move the properties to the html5.js instead?

Reply:
sorry, I thought those were duplicated in `w3c_dom2`, but in fact they are only defined on `HTMLDocument.prototype` ... so I put them back in `gecko_dom` for now.

Reply:
@andresdominguez are there still any problems after my last commit?